_plugins/scenario_tag.rb

```diff
@@ -126,4 +130,11 @@ def render_rouge(code, lang)
     result = formatter.format(lexer.lex(code))
     "<pre><code>#{result}</code></pre>"
   end
+
+  private
+
+  def request_with_file?(request)
+    values = request.values
+    values.size == 1 && values.first.respond_to?(:match?) && values.first.match?(/@\[.*\]/)
```

Comment:
This works for our current case, but seems brittle. Do we currently have a way to know in this context if, for example, it was a multipart request?